frontend/packages/client/src/components/ProposalInformation/ResultsPanel.js

```diff
@@ -0,0 +1,175 @@
+import { useMemo } from 'react';
+import { Svg } from '@cast/shared-components';
+import { Pill } from 'components';
+import { FilterValues } from 'const';
+import { parseDateFromServer } from 'utils';
+
+const Results = ({ voteResults, computedStatus }) => {
```

Comment:
This looks like we have two components in here - is it possible to separate these into separate component files?

Reply:
yes, makes sense. I just created a new component